docs/docs/pyproject.md

```diff
@@ -173,6 +173,17 @@ If a VCS is being used for a package, the exclude field will be seeded with the
 include = ["CHANGELOG.md"]
 ```
 
+You can also specify the formats for which these patterns have to be included, as shown here:
+
+```toml
+[tool.poetry]
+# ...
+include = [
```

Comment:
Might also want to note what the default behaviour here is, ie. if `format` is left out.